gradle.properties

```
@@ -1,8 +1,12 @@
 # Gradle
 org.gradle.jvmargs=-Xmx2048m -XX:MaxPermSize=512m
 
-# Jellyfin SDK
-sdk.useSnapshot=false
+# Allow switching between flavors of the Jellyfin SDK. Possible values are:
+# - "default"
+# - "local" (local Maven repository)
+# - "snapshot" (SDK master)
+# - "unstable-snapshot" (SDK with unstable Jellyfin API)
+sdk.version=local
```

This should be "default"

```suggestion
sdk.version=default
```